%\newcommand{\itab}[1]{\hspace{15mm}\ilap{#1}}
\chapter{Introduction}
\label{ch:intro}

The study of patterns is an innate human ability to create abstract representations for better perception, understanding, and interpretation of the world.  These patterns have no limitations on simplicity or complexity of the information they are trying to convey and offer a way to identify relationships among phenomena.  As datasets become increasingly complex and large, patterns become much more difficult to identify.  Human experts rely on computation tools to mimic human decision patterns via black-box models or simple linear regression tools.  In this thesis, we focus on imaging and genetic patterns, specifically dissecting complex relationships of somatic mutations and medical images of cancerous lesions, both separately and jointly.  By understanding the landscape of various patterns, we aim to improve precision medicine through early targeted treatment of cancer patients.  This has the potential to improve patient mortality rates, while simultaneously decreasing potent side effects of cancer treatments.

Medical imaging is omnipresent at initial and follow upstages of patient treatment to noninvasively visualize a cancerâ€™s appearance.  In conjunction with patient phenotypic symptoms, clinicians can     To finalize a treatment plan, clinicians rely on molecular profiling via invasive biopsy, however, it not an effective representation of the entire tumor due to the underlying spatial and temporal pathological heterogeneity, underestimating the molecular landscape of a tumor.   The efficacy of personalized treatment is therefore limited by intratumor genetic heterogeneity since treatments only target specific molecular aberrations.  To help mitigate this issue, clinicians have defined imaging phenotypes of a cancer's appearance and integrate the functional, molecular, and metabolic information to segment patient populations into phenotypic subsets that share similar prognoses and are likely to respond to similar therapies.  In the context of patterns, the classical example is tumour staging in which the involvement of adjacent tissues, regional lymph nodes and distant metastasis has an immediate and important impact on prognostic stratification and on the choice of therapeutic options. With increasingly powerful imaging tools, it is only natural to hypothesize, can imaging phenotypes be parlayed into further optimizing patient diagnosis.  Improved staging with the use of imaging might help in avoiding unnecessary surgical interventions, which increases the morbidity of disease without a substantial benefit in the reduction of mortality. On the other hand upstaging can lead to the choice of more aggressive therapies, which may reduce the recurrence rate.    

Through the lens of computer vision, imaging phenotypes are analogous to imaging features, which are a set of quantitative measurements to identify an object, in this case, the object is a tumor.  Classical methods of comparing medical images rely on the textural and spatial relationships between voxels and then use machine learning models to discriminate between tumors for numerous tasks such as segmentation, diagnosis, and prognosis tasks [citations] in various imaging modalities.  The same features, however, lack the resolution to answer more complex questions or are not applicable for other imaging modalities.  For example, consider the problem of predicting treatment response of a patient.  In such a scenario the discriminatory power of texture imaging features between follow-up images is too small to determine treatment response.  Similarly, Diffusion Weighted Magnetic Resonance Imaging (DWI) is a powerful tool that represents the underlying heterogeneity of tissue, but the voxels are continuous values and not graycale pixels, so texture imaging featuresare not applicable.  Motivated by more complex questions and new imaging tools, we explore and identify different imaging features that capture a larger amount of information showing applications to Positron Emission Tomography - Computed Tomography (PET-CT) and DWI.  

As clinicians incorporate phenotype and genotypes during patient treatment planning, it is imperative they also have access to relevant genetic biomarkers.  Why do we focus on somatic mutations and not the other abundant -omics ?  The lack of efficacy of treating the cancer comes from the heterogeneous accumulation of somatic mutations that result in a various types of cancer.  Moreover, even in the same type of cancer, there is little homogeneity between patients.  For example, within the most common hematologic cancer, diffuse large-B cell lymphoma (DLBCL), systemic therapy cures the majority of patients; however, a significant minority will succumb to the disease.  As such, there is a push to design treatments based on the molecular biology of the patient, by targeting genes that play a role in tumor angiogeneis.  Unfortunately, cancer heterogeneity, also has profound implications for drug therapy in cancer as the targeted therapy does not take into account the possibility of a low-frequency subpopulation harbouring a resistance.  For example, epidermal growth factor receptor (EGFR) is a cell surface receptor that plays a critical role in cell proliferation, survival, and differentiation and is tightly regulated by various extracelluar ligands in healthy cells. Targeted treatments of inhibiting EGFR has shown great promise in lung, kidney, and colorectal cancers, but unfortunately only a fraction are responders and some are resistant or acquire resistance to the therapy.  These resistances occur as part of the functional redundancy that is inherent in molecular biology where multiple genes play a role in the signaling pathway.  In the case of the EGFR gene, mutually exclusively mutations in KRAS confer resistance to to EGFR-targeted treatment.  Thus methods to identify co-occurrence of mutations can aid in identifying patients that will respond to the treatment, as well as identifying potential mutations for targeted treatment.  The sparse and high dimensionality of somatic mutation data, however, precludes us from using simple statistical models.  Considering a binary representation for the dataset, in DLBCL alone, there are at least 5000 somatic mutations, therefore $2^{n}$ permutations are possible.  Naively one can increase the number of parameters to fit a linear model, but that becomes computational intractable.  We consider the difficulty of learning the complex underlying structure of mutations and create a model to describe co-occurring mutations regardless of their frequency using models based on the Beta-Bernoulli Process (BeBP).  These co-occurring mutations can then be used as genetic biomarkers and other downstream tasks such as further patient subtyping.

In the problems discussed so far, we have assumed the different modalities of cancer data are only studied concurrently.  However, as cancer is a result of a combination of genetic instabilities and epigentic alterations, many integration based models have become popular for analyzing genetic datasets.  For example, one can predict breast cancer in a patient by identifying DNA sequence variation through association studies in family- and population-based data, and then integrating it with molecular pathway information [citation].  Extensions to integrate imaging and genetic data has gained traction in a nascent topic called radiogenomics or imaging genomics, but the structure and distribution of data generated by different modalities makes it a very challenging task. Many studies have focused on prediction of a few frequent genetic markers or a large set of genetic markers from imaging features, which limits the applicability of models to very specific tasks. Our proposed method relates the set modalities follows the paradigm that is often seen image to text or text to image generation.  To account for the complexity and sparsity of in the medical and biology domain we utilize paired variational auto-encoders with conditional normalizing flow priors that encourages the datasets to share similar subsets of the large set of possible behaviors.  We can use this shared space to predict patients' mutation profile based on their corresponding medical image.  The benefits of this model are twofold: patient mortality is best reduced when the disease is targeted during its early stages where the mutational landscape within a tumor is still relatively homogeneous and cancer spread is limited and we may uncover interesting relationships between the two vastly different datasets.

\section{Thesis Organization}

We now provide an overview of the contributions of each chapter, including methodologies and results, as well as an overview of the chapter structure.  Since we address a number of different problems in subsequent chapters, we provide more through literature reviews of prior relevant work for each of the models.

\subsection{Background}

We begin by reviewing many of the imaging and statistical concepts that are utilized throughout this thesis. The chapter starts with an overview of current image feature generation techniques for two and three dimensional images.  This will serve as a backbone to the reasoning of why we choose specific imaging features for our tasks.

We then describe methods in probabilistic models and their use in modeling sparse high-dimensional data by presenting latent feature models. Together, these concepts enable examination of the importance of prior distributions, the distribution of the unknown parameters, and the posterior distribution, the updated parameters given the data.  As exact inference is infeasible for the relevant parameters, the chapter then moves onto Bayesian inference techniques specifically amortized inference and variational inference. We conclude the chapter with background material on the most common stochastic processes we use in developing our Bayesian nonparametric models: the Dirichlet process and the beta process.


\subsection{Contributions}

The original contributions of this thesis follow in the remaining chapters, where chapter 3 introduces 3D imaging features and histogram features for treatment response prediction and tumor prediction respectively.   Thorough empirical examination of datasets that incorporate images from PET/CT and DWI we show our model results in very competitive predictive and interpretative performance. In Chapter 4, we introduce a framework for determining correlated sets of co-occurring somatic mutations.  Using datasets from The Cancer Genomic Archive (TCGA) we show we can generate biologically interpretable co-occurring mutations for patient subtyping and targeted treatment.  In Chapter 5, we introduce a framework for mapping somatic mutations and 3D imaging features of tumors into the same subspace for predicting somatic mutations from a patient's corresponding tumor image.  To show predictive performance of our model, we create a dataset of 3D point cloud representations of tumors from a subset of the The Cancer Imaging Archive data that has corresponding somatic mutation.  

